Gemfile.lock

```diff
@@ -23,6 +23,7 @@ PATH
   specs:
     synthea (1.0.0)
       area
+      chunky_png
```

Comment:
No, the change is in the gemspec which is probably the proper place for this, since it is a runtime dependency for the gem.